src/test/java/com/medeiros/RoostTest/openRestaurantTablesGetTest.java

```diff
@@ -61,12 +61,12 @@ public void openRestaurantTablesGet_Test() {
                 if (response.statusCode() == 200) {
 					System.out.println("Description: Return All Open Restaurant Tables from a Point of Sale");
       
-              if (response.jsonPath().get("id") != null) {  
-                MatcherAssert.assertThat(response.jsonPath().get("id"), instanceOf(Integer.class));  
+              if (response.jsonPath().getList("id").get(0) != null) {  
+                MatcherAssert.assertThat(response.jsonPath().getList("id").get(0), instanceOf(Integer.class));  
           }
       
-              if (response.jsonPath().get("restaurant_table_id") != null) {  
-                MatcherAssert.assertThat(response.jsonPath().get("restaurant_table_id"), instanceOf(Integer.class));  
+              if (response.jsonPath().getList("restaurant_table_id").get(0) != null) {  
+                MatcherAssert.assertThat(response.jsonPath().getList("restaurant_table_id").get(0), instanceOf(Integer.class));  
```

Comment:
**ERROR**
```
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.89 s <<< FAILURE! - in com.medeiros.RoostTest.openRestaurantTablesGetTest
[ERROR] openRestaurantTablesGet_Test  Time elapsed: 0.864 s  <<< FAILURE!
java.lang.AssertionError:

Expected: an instance of java.lang.Integer
     but: <[0]> is a java.util.ArrayList
        at com.medeiros.RoostTest.openRestaurantTablesGetTest.openRestaurantTablesGet_Test(openRestaurantTablesGetTest.java:65)
```

**RESPONSE**
`[NEGOTIATOR] ℹ  info      HTTP response is: {"statusCode":200,"headers":{"Content-type":"application/json"},"body":[{"id":0,"restaurant_table_id":0}]}`

The response we are getting is an array, while the code is trying to access it's value like an object.

**FIX** :- Changed the code so that the response is correctly traversed.